tests/checks/integration/test_rabbitmq.py

```diff
@@ -103,3 +105,16 @@ def test_queue_regex(self):
         self.assertServiceCheckOK('rabbitmq.aliveness', tags=['vhost:/'])
 
         self.coverage_report()
+
+    def test_family_tagging(self):
+        self.run_check(CONFIG_WITH_FAMILY)
+
+        # Node attributes
+        for mname in COMMON_METRICS:
+            self.assertMetricTagPrefix(mname, 'rabbitmq_node', count=1)
+
+        for mname in Q_METRICS:
+            self.assertMetricTag('rabbitmq.queue.%s' %
+                                 mname, 'rabbitmq_queue_family:test', count=2)
+
+        self.coverage_report()
```

Comment:
There's one `assert` that's missing to have full coverage:

```
self.assertServiceCheckOK('rabbitmq.aliveness', tags=['vhost:/'])
```

(see https://travis-ci.org/DataDog/dd-agent/jobs/120730540#L390-L397)